ElectronClient/app/app.js

```diff
@@ -500,6 +503,45 @@ class Application extends BaseApplication {
 			screens: ['Main'],
 		});
 
+		templateItems.push({
+			label: _('Create note from template'),
+			visible: fs.pathExistsSync(Setting.value('profileDir') + '/templates'),
```

Comment:
I think this whole block should not be executed (templateItems remains an empty array) if the templates folder does not exist. And we should simply hide or show the root template inside of each individual item. Also please use `shim.fsDriver().exists` (async) to test for the existence of a file.

Reply:
Edit: Sorry I missed that "Open template directory" should always be visible. I guess if you could just cache the value of `shim.fsDriver().exists(Setting.value('profileDir') + '/templates')` that would be enough. That way the check won't be repeated for each menu item.

Reply:
Done! I had to make the entire function async.